<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformWidget</class>
 <widget class="QWidget" name="TransformWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>错切：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zoomXEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="zoomYEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="ensureBtn">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>缩放：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scaleXEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scaleYEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>位置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="positionXEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="positionYEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>旋转：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rotateEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Transform Window:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QWidget" name="widget_6" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteAllBtn">
        <property name="text">
         <string>删除所有</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="unFillBtn">
        <property name="text">
         <string>取消着色</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ensureBtn</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>reciveData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteBtn</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>deletePolygon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteAllBtn</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>deleteAllPolygon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>unFillBtn</sender>
   <signal>clicked()</signal>
   <receiver>TransformWidget</receiver>
   <slot>cancleFill()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>424</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reciveData()</slot>
  <slot>deletePolygon()</slot>
  <slot>deleteAllPolygon()</slot>
  <slot>cancleFill()</slot>
 </slots>
</ui>
